### ISSUE TITLE
Please update for Foundry v10

### ISSUE THREAD

**Issue by BigTug** [kckaiwei/TurnMarker-alt#108]
Installed 2.9.0 no longer allowed to run on foundry startup and removed from Add-on Modules list with v10 installed.
Also can no longer be installed from Foundry Install Module page.
Error received on load:
setup.js:141 
        
       [turnmarker] Metadata validation failed for module "turnmarker": The file "css/turnmarker.css" included by module turnmarker does not exist

**Comment by JonWaterfall** [comment]
There's a pull request to fix this, but it seems that @kckaiwei isn't active to review it.
https://github.com/kckaiwei/TurnMarker-alt/pull/107

**Comment by RuebenMF** [comment]
How do we properly uninstall this module? I've installed it and now i get the same error @BigTug mentioned - but it's not in my list of installed modules to uninstall.
I deleted the module folder from data>modules but i'm still getting:

`setup.js:141 [turnmarker] Metadata validation failed for module "turnmarker": The file "css/turnmarker.css" included by module turnmarker does not exist`

Is there a manifest file i need to remove it from? if so where is it?

**Comment by vitrifiedglass** [comment]
@RuebenMF Quite late here, but is it possible that one of your worlds is still trying to use that module? I removed my old campaign worlds (to a safe place) in addition to deleting the module folder and it resolved the issue for me.

**Comment by coffiarts** [comment]
@vitrifiedglass  What do you mean exactly by "removing your world to a safe place"? Did you create a backup and then (after deleting the TM folder) re-import it to your world list in foundry? Or do you mean by "old" that it is a world that you are not playing anymore?
Because I am having the exact same issue as described by @RuebenMF. I can neither install nor uninstall the TM module, but my world keeps throwing the quoted error message.
My world is still active, I need it up and running. So I am searching for a way to cleanly remove the buggy TM module, without breaking it.

**Comment by vitrifiedglass** [comment]
@coffiarts I mean that I created a backup of my old worlds that I stopped playing (just in case) then removed them from my world directory. I do not add them again.

Based on what you posted in that other thread, I suspect your LightHUD module is trying to do something with the tile used by Turnmarker to show whose turn it is. Have you tried deleting the actual Turnmarker files in your operating system, instead of with Foundry? Does the bug go away if you uninstall LightHUD?

**Comment by vitrifiedglass** [comment]
From Foundry Discord: "Go to the data directory of Foundry. In the subfolder /modules should be a folder called "turnmarker" or similar. Delete that, restart Foundry." Please try this.

**Comment by coffiarts** [comment]
@vitrifiedglass Yes, thanks, I could now get rid of it by simply deleting the modules/turnmarker directory. And the LightsHUD error ist gone as well.

**Comment by illandril** [comment]
For any late v10 upgraders who come across this issue, here are two alternative modules that provide similar functionality:
- [Combat Booster](https://foundryvtt.com/packages/combatbooster)
- [Illandril's Turn Marker](https://foundryvtt.com/packages/package/illandril-turn-marker)
